Automattic/jetpack
# Social Preview: add to pre-publish sidebar

Fixes https://github.com/Automattic/jetpack/issues/19915

#### Changes proposed in this Pull Request:

This PR adds the Jetpack Social Preview panel to the pre-publish sidebar.

<img width="1219" alt="Screen Shot 2021-06-01 at 12 32 19 pm" src="https://user-images.githubusercontent.com/6458278/120259428-4a957c80-c2d7-11eb-873e-c34776bd66b6.png">

The Social Preview panel still appears in the Jetpack sidebar. (Or should we remove it from here?)

<img width="1220" alt="Screen Shot 2021-06-01 at 12 36 19 pm" src="https://user-images.githubusercontent.com/6458278/120259435-4cf7d680-c2d7-11eb-977d-1b3f0d89e896.png">


#### Jetpack product discussion
p1HpG7-c4u-p2

#### Does this pull request change what data or activity we track or use?
Nah.

#### Testing instructions

1. Run locally and create a new post.
2. Check that the social preview appears (and the preview modal works) in the Jetpack menu, and in the pre-publish sidebar.
3. Apply `D62094-code` to your sandbox, and check that the panel displays the upgrade nudge for a free, sandboxed site.
4. Upgrade your test site to a Business plan. The Social Preview panel should show the preview modal.


<img width="264" alt="Screen Shot 2021-06-01 at 12 54 23 pm" src="https://user-images.githubusercontent.com/6458278/120260087-7fee9a00-c2d8-11eb-8f38-2894eb9e7d43.png">
